internal/scraper/scraper_N_test.go

```diff
@@ -7,6 +7,61 @@ import (
 
 func TestScraper_N(t *testing.T) {
 	testcases := []testcase{
+		{
```

Comment:
Excellent! In your PR description, you also ticked `chatelaine.com`, `downshiftology.com`, and `wellplated.com`. Since we are missing a test for `chatelaine.com`, could you please add one?